GuyaTH/GT001.xml

```diff
@@ -0,0 +1,126 @@
+                    multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/">This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0011GuyaTH"/>
+                        <idno facs="GT/001/" n="150">GT-001</idno>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <title>Taʾāmmǝra Māryām (ʾAkkonu Bǝʾsi type ) </title>
+                            <textLang xml:lang="gez"/>
+                            <colophon xml:id="coloph1"><locus target="#439"/></colophon>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/></support>
+                                <extent>
+                                    <measure unit="leaf">145</measure>
+                                    <dimensions type="outer">
+                                        <height unit="mm">280</height>
+                                        <width unit="mm">220</width>
+                                    </dimensions>
+                                </extent>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="14"/>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic"/>
+                        </handDesc>
+                        <decoDesc>
+                            <decoNote xml:id="d1"><desc>decorated in the so-called Second Gondarine style decoration
+                            </desc></decoNote>
+                            <decoNote xml:id="d2"></decoNote>
+                            <decoNote xml:id="d3"></decoNote>
+                            <decoNote xml:id="d4"></decoNote>
+                            <decoNote xml:id="d5"></decoNote>
+                            
+                        </decoDesc>
+                        <additions><list>
+                                <item xml:id="a1"><desc>the manuscript contains short Ge'ez notes provided as colophons and landmark notes.
+                                    </desc></item>
+                            <item xml:id="a2"><desc>the manuscript contains short Greek,
```

Comment:
Really Greek? If so, to be added among the languages

Reply:
Not Greek, there is another issue, of "cryptic writing" (appeared in the field report)